review_practice/react-app/src/App.js

```diff
@@ -0,0 +1,32 @@
+// Import React modules
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import { ApolloProvider } from '@apollo/react-hooks';
+
+// Import pages
+import { SearchRoom } from 'pages/SearchRoom';
+import Login from 'pages/Login';
+import RegisterUser from 'pages/RegisterUser';
```

Comment:
```suggestion
import { SearchRoom, Login, RegisterUser } from 'pages';
```

이와 같이 import 할 수 있도록 react-app/src/pages 하단에 index.js 파일에서 각 page를 import 하는 방식은 어떤가요? 

또한 명확하게 `pages`에서 각 컴포넌트들을 import 하는 것을 알 수 있기에 `// Import pages`는 불필요한 주석이라고 생각되는데, 어떻게 생각하시나요? 🤔